provisioning/transport/http/src/Generated/RuntimeRegistration.cs

```diff
@@ -1,11 +1,6 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.Azure.Devices.Provisioning.Client.Transport.Models;
```

Comment:
Most of these changes are just code cleanup (like variable renames, etc.) but I'll point out more interesting changes.

Nothing is necessary for this bug fix, though.